src/libraries/System.Resources.Extensions/src/BinaryFormat/BinaryFormattedObject.IParseState.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.IO;
+using System.Runtime.Serialization.BinaryFormat;
+
+namespace System.Resources.Extensions.BinaryFormat;
+
+internal sealed partial class BinaryFormattedObject
```

Comment:
For this, and every other file in this project:

This library uses the standard dotnet/runtime managed libraries pathing convention

`[lib]/src/Full/Namespace/That/The/Type/Is/In/File.cs`

All of these files under `src/BinaryFormat` need to move to `src/System/Resources/Extensions/BinaryFormat/` (assuming they're all in that namespace).